src/services/mondomarket/index.jsx

```
@@ -0,0 +1,238 @@
+      {/* Offer Modal  */}
+      <Modal
+        title="Your Offer"
+        isOpen={offerModal}
+        onClose={() => {
+          setOfferModal(false);
+          setAmount("");
+        }}
+      >
+        <div className="flex gap-5 items-center mb-10">
+          <img
+            src={mondo?.images[0]?.image}
```

ESLint

[error] 'mondo.images' is missing in props validation

(react/prop-types)